### DISCUSSION TITLE
simulation Fault stops early

### DISCUSSION THREAD

**Discussion by IgorrrEgorrr** [CURENT/andes#539] [Modeling]
Hello. 
The package is useful and with good docs_andes.
In my own model consisting of 21 buses, slach, pv and loads, I have added List “Fault”
Parameters List “Fault” 
uid idx u name bus tf tc xf rf
0 1 1 1 Fault_1 9 1,2 1,4 0,1 0,1 0,1

The model counts ss.PFlow.run() correctly.
I get the following error:
I enter a simulation time interval of 5 sec
ss.TDS.config.tf = 5
However, ss.TDS.run() - the model will calculate only up to the start of Fault tf=1.2 sec. 
Note that the names of "tf" parameters TDS.config.tf and Fault tf are the same. 
Please help me to correct the error
I have attached my model .xlsx 
[ss_sigre_lv_solar_fault_ivt03.xlsx](https://github.com/CURENT/andes/files/15146464/ss_sigre_lv_solar_fault_ivt03.xlsx)

**Comment by IgorrrEgorrr** [comment]
I found out that the error due to joining solar.

**Reply by IgorrrEgorrr** [reply]
Correctly run case with the addition of the static PQ generator and fault. Can you show an example of converting a PQ generator to solar? To understand the calculation of the solar initial conditions on an example.

**Reply by cuihantao** [reply]
The solar model has different power output characteristics when voltage is low. You can try a mild fault (set `xf` to a larger value, say `1` or `0.1`) and see if it solves. Then, gradually decrease `xf` and adjust solar control parameters.

**Reply by IgorrrEgorrr** [reply]
Thank you. You correctly suggested to reduce the fault power. Can you recommend a textbook with a detailed description of the use of Andes?

**Reply by jinningwang** [reply]
There are example notebooks in the documentation.
We also have a repository of course project as a hands-on tutorial. https://github.com/CURENT/ece522

**Reply by IgorrrEgorrr** [reply]
I analyze my solar - Bus, PQ, Line, Slack,REGCA1, REPCA1,REECA1

 ss = andes.run(....)
got
ss.Bus.v.v=[....]
ss.Bus.a.v=[....]

ss.PFlow.run()    
got
print(ss.Bus.v.v)
print(ss.Bus.a.v)
ss.exit_code= 0

When running ss.TDS.init(), I got the error
Suspect initialization issue! Simulation may crash!

  Name | Var. Value | Eqn. Mismatch
----------+------------+--------------
 a Bus 1 | 0.000 | 0.004        
 a Bus 17 | -0.036 | -0.004       
 v Bus 1 | 1.000 | 0.002        
 v Bus 17 | 0.944 | -0.002 

How to change the initial conditions for the next step ss.TDS.run() ??

**Reply by IgorrrEgorrr** [reply]
Thank you for hands-on tutorial.

**Reply by IgorrrEgorrr** [reply]
[ss_sigre_lv_solar_fault_ivt04_v7.xlsx](https://github.com/CURENT/andes/files/15179711/ss_sigre_lv_solar_fault_ivt04_v7.xlsx)
The model calculates without errors ss = andes.run(get_case,...), ss.PFlow.run(), ss.exit_code=0.
Error at ss.TDS.init().
Question. Where are the values ​​of column 3 ("Eqn. Mismatch"), how to read them, how are they calculated.
Are all parameters from the table wrong (column 2 "Var. Value")? How can "V bus1"=1 and "a Buys 1" =0 be out of range when it is slack? Please advise, what are the practical steps to configure the parameters of my scheme to fulfill the conditions of ss.TDS.init()? This is a very useful  pandapower SIGRE LV case.
